keyboards/westberry/common/readme.md

```diff
@@ -0,0 +1,8 @@
+## Description
+On a limited hardware budget, you can use WB32's PWM Channels + some mos tubes to drive your rgb 
```

Comment:
So this is using a bunch of pins to drive a bunch of dumb rgb leds? 

Namely, more information about what this is doing and why would be very useful here.